isnad.csl

```diff
@@ -0,0 +1,1387 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" page-range-format="expanded">
+  <!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title xml:lang="tr">İSNAD</title>
+    <title-short xml:lang="tr">İAS</title-short>
+    <id>http://www.zotero.org/styles/isnad</id>
+    <link href="http://www.zotero.org/styles/isnad" rel="self" xml:lang="tr"/>
+    <link href="http://www.isnadsistemi.org/" rel="documentation"/>
+    <author>
+      <name>Muhammet Tarakçı</name>
+      <email>muhammettarakci@gmail.com</email>
+    </author>
+    <category citation-format="note"/>
+    <category field="generic-base"/>
+    <summary>The common citation style created by Dimer (http://dimer.cumhuriyet.edu.tr) for academic journals published in the field of humanities and social sciences in Turkey. Türkiye'de beşeri ve sosyal bilimler alanlarında yayın yapan dergiler için Dimer (http://dimer.cumhuriyet.edu.tr) tarafından oluşturulmuş ortak atıf şablonu.</summary>
+    <updated>2018-03-31T21:45:33+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="tr">
+    <terms>
+      <term name="page-range-delimiter">-</term>
+      <term name="in"/>
+      <term name="et-al">v.dğr.</term>
+      <term name="et-al" form="short">v.dğr.</term>
+      <term name="and others">v.dğr.</term>
+      <term name="and others" form="short">v.dğr.</term>
+      <term name="issue" form="short">sy.</term>
+      <term name="translator" form="short">trc.</term>
+      <term name="translator" form="verb-short">trc.</term>
+      <term name="and"> - </term>
```

Comment:
I don't think this is right -- you don't actually want to define "and" as a hyphen. To get something like
> Topaloğlu, Bekir - Yavuz, Y. Şevki - Çelebi, İlyas.

you'd just do

```
<names variable="author">
  <name delimiter=" - " delimiter-precedes-last="always" name-as-sort-order="all"/>
</name>
```